source/admin_manual/list_of_events.rst

```diff
@@ -2726,6 +2726,94 @@ Event emitted when dict server finishes transaction. Same fields as
 :ref:`dict_transaction_finished`.
 
 
+Pre-login Client
+================
+
+.. _pre_login_client:
+
+Client
+------
+
+Common fields:
+
++---------------------+------------------------------------------------------+
+| Field               | Description                                          |
++=====================+======================================================+
+| local_ip            | Local IP address                                     |
++---------------------+------------------------------------------------------+
+| local_port          | Local port                                           |
++---------------------+------------------------------------------------------+
+| remote_ip           | Remote IP address                                    |
++---------------------+------------------------------------------------------+
+| remote_port         | Remote port                                          |
++---------------------+------------------------------------------------------+
+| user                | Full username                                        |
++---------------------+------------------------------------------------------+
+| service             | Name of service e.g. ``submission``, ``imap``        |
++---------------------+------------------------------------------------------+
+
+
+Login proxy
+===========
+
+.. versionadded:: v2.3.18
+
+Emitted when login processes proxy a user connection to a backend.
```

Comment:
This sentence needs fixing somehow. Maybe "Emitted when login process proxies a connection to a backend."

Reply:
Changed.